external/fixedWidth/src/main/scala/za/co/absa/enceladus/fixedWidth/types/SparkTypeResolver.scala

```diff
@@ -56,7 +57,7 @@ object SparkTypeResolver {
 
   private def castToDateTimeType(field: StructField, dateString: String, pattern: String): Any = {
     try {
-      val parser: EnceladusDateParser = EnceladusDateParser(pattern)
+      val parser: EnceladusDateTimeParser = EnceladusDateTimeParser(pattern)
```

Comment:
As you surely noticed I like and tend to have my things explicitly typed. Sure a habit from other languages.
Is it a problem in our code? I would have to conscientiously concentrate to avoid the usage.

Reply:
I get that but I would prefer a consistent style and very much like type inference, especially in cases like this.

Reply:
OK, I will try to avoid type declarations on direct (case) class or primitive types assignment. 😄

Reply:
Thanks!